services/movie.py

```diff
@@ -0,0 +1,45 @@
+from typing import Optional
+
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(genres_ids: str = None, actors_ids: str = None) -> None:
```

Comment:
If an argument can be `None`  you must show it in a type hint. Fix everywhere
```suggestion
def get_movies(genres_ids: str | None = None, actors_ids: str | None = None) -> None:
```